Answers.md

```diff
@@ -0,0 +1,23 @@
+1) If you had to describe semantic HTML to the next cohort of students, what would you say?
+
+HTML is the foundation of how a website can look. HTML provides a way to display data onto
+a page in any order you would like.
+
```

Comment:
yes, but what is semantic HTML? 

From [W3Schools.com](https://www.w3schools.com/html/html5_semantic_elements.asp): 

"A semantic element clearly describes its meaning to both the browser and the developer."
"__Semantic elements__ clearly express what the content's function is." 

Examples would be p, article, caption, etc. Non-semantic elements include div and span>. These don't expressly tell you what the function of the content is.